tcms/rpc/tests/test_testcase.py

```diff
@@ -569,14 +741,19 @@ def verify_api_with_permission(self):
         self.assertEqual(result[str(self.case_2.pk)], 15)
         self.assertEqual(result[str(self.case_3.pk)], 25)
 
+        # Test query None
+        result = self.rpc_client.TestCase.sortkeys()
+        self.assertIsNotNone(result)
+        self.assertEqual(len(result), 3)
```

Comment:
Same comment as above - will break the moment there are more objects into the database. Check if the list is not empty instead.